src/Components/SubscriptionEditor/SubscriptionEditor.jsx

```diff
@@ -79,14 +80,27 @@ export default class SubscriptionEditor extends React.Component<Props> {
                                 value={subscription.tags}
                                 onChange={nextTags => {
                                     onChange({
-                                        tags: nextTags,
+                                        tags: subscription.any_tags ? null : nextTags,
                                     });
                                 }}
                                 availableTags={tags}
+                                isDisabled={subscription.any_tags}
                             />
                         </ValidationWrapperV1>
                     </div>
                 </div>
+                <div className={cn("row")}>
+                    <span className={cn("any_tags")}>
+                        <Toggle
+                            onChange={checked =>
+                                onChange({
+                                    any_tags: checked,
```

Comment:
If you put new prop and then change it, you must describe it in parrent state types. I don't see it

Reply:
How to do it?